content/guides/models/registry/configure_registry.md

```diff
@@ -7,107 +7,121 @@ title: Configure registry access
+| Create a new custom registry                                   | Create           |        |   X    |   X   |
+| Edit collection card (description)                             | Update           |        |   X    |   X   |
+| Edit linked artifact description                               | Update           |        |   X    |   X   |
+| Add or delete a collection’s tag                               | Update           |        |   X    |   X   |
+| Add or delete an alias from a linked artifact                  | Update           |        |   X    |   X   |
+| Link a new artifact                                            | Update           |        |   X    |   X   |
+| Edit allowed types list for a registry                         | Update           |        |   X    |   X   |
+| Edit custom registry name                                      | Update           |        |   X    |   X   |
+| Delete a collection                                            | Delete           |        |   X    |   X   |
+| Delete an automation                                           | Delete           |        |   X    |   X   |
+| Unlink an artifact from a registry                             | Delete           |        |   X    |   X   |
+| Edit accepted artifact types for a registry                    | Admin            |        |        |   X   |
+| Change registry visibility (Organization or Restricted)        | Admin            |        |        |   X   |
+| Add users to a registry                                        | Admin            |        |        |   X   |
+| Assign or change a user's role in a registry                   | Admin            |        |        |   X   |
 
-{{< img src="/images/registry/configure_role_registry.gif" alt="" >}}
-
-## Remove a user from a registry
-1. Navigate to the **Registry** App in the W&B App UI.
-2. Select a core or custom registry.
-3. Click on the gear icon on the upper right hand corner.
-4. Scroll to the **Registry members and roles** section and type in the username of the member you want to remove.
-5. Click the **Delete** button.
 
+### Inherited permissions
 
+A user's permission in a registry depends on the highest level of privilege assigned to that user, whether through a team or individual. 
 
-## Registry visibility types
+For example, suppose a registry admin adds a user called Nico to Registry A and assigns them a `Viewer` registry role. The registry admin then adds a team called Foundation Model Team to Registry A and assigns Foundation Model Team a `Member` registry role. Nico is a member of Foundation Model Team.
 
-There are two registry visibility types: restricted or organization visibility. The following table describes who has access to the registry by default:
+Since Nico is a member of Foundation Model Team, W&B grants Nico `Member` registry role permissions because `Member` registry roles have higher level of permissions than `Viewer`. 
```

Comment:
```suggestion
Nico is a member of the Foundation Model Team, which is a **Member** of the Registry. Because **Member** has more permission than **Viewer**, W&B grants Nico the **Member** role.
```